client/service/src/lib.rs

```diff
@@ -335,7 +335,8 @@ fn build_network_future<
 
 		// We poll `imported_blocks_stream`.
 		while let Poll::Ready(Some(notification)) = Pin::new(&mut imported_blocks_stream).poll_next(cx) {
-			network.on_block_imported(notification.header, Vec::new(), notification.is_new_best);
+			network.on_block_imported(notification.header, Vec::new(), notification.is_new_best, false);
+			network.service().announce_block(notification.hash, Vec::new());
```

Comment:
Well, you can pass `true` here.
The entire point of passing an extra boolean parameter is that you don't have to call `announce_block` separately.

```suggestion
			network.on_block_imported(notification.header, Vec::new(), notification.is_new_best, true);
```

Reply:
Although maybe not having a boolean is better?
If you're reading the code of sc_service trying to figure out when blocks are announced, this little extra `true` is easy to miss.

Reply:
`enum Announce { Yes, No }`

Reply:
I finally get it :sweat_smile: calling the code explicitly is more visible than the boolean